lib/cc/cli/help.rb

```diff
@@ -14,7 +14,7 @@ def run
 
       def commands
         [
-          "analyze [-f format]",
+          "analyze [-f format] [-e engine] <path>",
```

Comment:
yeah, agreed.

At first I was curious about whether we have `--upgrade` in the help commands, and noticed this absence as well.